### PR TITLE
Taylorr/feature/updater pipemode

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: src/groundStation/groundStation.py [hunk 1]
```diff
@@ -316,13 +312,10 @@ def receive(self):
                 print(rxData)
 
     def handlePipeMode(self):
-        if self.uTrns_enable == True:
-            if (time.time() - self.uTrns.last_tx_time) > self.uTrns.pipetimeout_s:
-                self.uTrns.enterPipeMode()
-                #may need to add delay here?
-                command, port, toSend = self.getInput(inVal= self.satellite +'.general.UHF_IS_IN_PIPE_NOTIFICATION(1)')
+        if self.uTrns is not None:
+            if self.uTrns.handlePipeMode():
```
Comment (joshdellaz) [inline src/groundStation/groundStation.py:316]: This and the next line are removed as per tganley/bugfix/flatsat-using-sdr. @tganley can you open a PR for that asap?

File: src/groundStation/uTransceiver.py [hunk 2]
```diff
@@ -77,54 +76,57 @@ def listen_zmq(self):
 
         socket.disconnect("tcp://localhost:%s" % self.rxport)
 
+    def handlePipeMode(self):
+        if (time.time() - self.last_tx_time) > self.pipetimeout_s:
+            self.enterPipeMode()
+            return True
```
Comment (joshdellaz) [inline src/groundStation/uTransceiver.py:82]: need to add self.last_tx_time = time.time() before this return True

File: src/groundStation/groundStation.py (hunk 1 above)
Comment (Taylorr82) [reply src/groundStation/groundStation.py:316]: We seem to be slipping back to the old ways with the flatsat-testing branches...

Comment (Taylorr82) [reply src/groundStation/groundStation.py:316]: When I figure out why this is failing tomorrow I'll merge this branch with this change anyway. These long lived branches with "little" fixes all over the place are difficult to work with

File: src/groundStation/uTransceiver.py (hunk 2 above)
Comment (Taylorr82) [reply src/groundStation/uTransceiver.py:82]: Ok

**Pull request merged**